package.json

```diff
@@ -48,15 +48,16 @@
   "devDependencies": {
     "@kadira/storybook": "2.20.1",
     "babel-eslint": "7.0.0",
+    "babel-polyfill": "6.16.0",
     "babel-preset-es2015": "6.16.0",
     "babel-preset-react": "6.16.0",
     "babel-preset-stage-0": "6.16.0",
     "babelify": "7.3.0",
     "browserify": "13.1.0",
     "chai": "3.5.0",
     "enzyme": "2.4.1",
-    "eslint-plugin-react": "6.3.0",
-    "flow-bin": "0.33.0",
+    "eslint-plugin-react": "5.2.2",
+    "flow-bin": "0.27.0",
```

Comment:
A large number of version changes were made recently without updating shrink. Once I fixed shrink I found 20+ new lint errors and 10 or so flow errors. Fixing the other shrink problems has been quite time consuming... I would like to avoid piling on more. Feels like running on a treadmill. If someone is passionate about upgrading flow or eslint I am happy to let them wade through it. :)